buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -178,51 +178,60 @@ public static int hashCode(ByteBuf buffer) {
 
     /**
      * Returns {@code true} if and only if the two specified buffers are
-     * identical to each other as described in {@code ChannelBuffer#equals(Object)}.
-     * This method is useful when implementing a new buffer type.
+     * identical to each other for {@code len} bytes starting at {@code iLhs}
```

Comment:
Let me expand these out...

lhs = leftHandSide
rhs = rightHandSide
iLhs = indexLeftHandSide
iRhs = indexRightHandSide
len = length

For `iLhs` and `iRhs` it may be better to call them `startIndex`...WDYT?